src/app/components/AutoComplete.svelte

```diff
@@ -165,7 +174,11 @@
 <svelte:window on:keydown={handleKeydown} />
 <div class="autocomplete mb-m">
   {#if label !== ''}
-    <p class="autocomplete__label">{label}</p>
+    {#if bold}
+      <p class="autocomplete__label">{label}</p>
+    {:else}
+      <p class="autocomplete__label__normal">{label}</p>
+    {/if}
```

Comment:
I will refactor this in a future PR, but there's a way of tidying this up. Basically, there should be a `bold` modifier, and then that modifier can be toggled with  the `bold` variable.

See: https://svelte.dev/tutorial/classes